docs/FAQ/identity_management_FAQ.md

```diff
@@ -1,18 +1,18 @@
 ## Identity Management (Membership Service)
 &nbsp;
 ##### What is unique about the fabric's Membership Service module?
-The design and implementation of the Membership Service module encompass many of the latest advances in cryptography, which we believe make it stand apart from other alternatives.
+One of the things that makes the Membership Service module stand out from the pack is our implementation of the latest advances in cryptography.
```

Comment:
Well it's not for me to say but I doubt a long digression into competing versions of membership service is what anyone wants to see in that FAQ. 

Is it possible to change the wording of the question? Because I'm just not sure how you answer

"What is unique about the fabric's Membership Service module?" 

Without drawing a comparison, even if it's an implicit one. 

If the question was worded:

"What are some of the security features of the fabric's Membership Service module?" 

You could provide exactly the same information in the answer without the need to make a comparison.